helm_deploy/prison-to-nhs-update/templates/_envs.tpl

```diff
@@ -54,13 +54,13 @@ env:
         name: {{ template "app.name" . }}
         key: PRISON_TO_NHS_CLIENT_SECRET
 
-  - name: SQS_AWS_ACCESS_KEY_ID
+  - name: SQS_QUEUE_ACCESS_KEY_ID
```

Comment:
change env var to match property and then we don't need the extra mapping configuration in application.yml